lib/firstLetters.js

```diff
@@ -29,13 +29,13 @@ const vowelFirstLetters = {
  * @example <caption>English first letters</caption>
  * firstLetters('sabad marai. so mar rahai; fir. marai na, doojee vaar |') // => sm.smr;f.mn,dv|
  */
-const firstLetters = line => line
+const firstLetters = ( line ) => line
```

Comment:
Something is definitely wrong with ESLint, it modified all the files. This isn't proper as per the rules we follow.

Reply:
I think this rule is missing and is causing that 
```json
 "arrow-parens": [
     "error",
     "as-needed",
     {
       "requireForBlockBody": false
     }
  ],
```